.github/workflows/actions.yml

```
@@ -0,0 +1,44 @@
+name: Continuous Integration
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+jobs:
+  specs:
+    runs-on: zendesk-stable
```

```suggestion
    runs-on: ubuntu-latest
```

Any reason not to use a standard runner here?